lib/gis/testsuite/test_parser_json.py

```diff
@@ -50,7 +52,9 @@ def test_r_slope_aspect_json(self):
             },
         ]
 
-        stdout, stderr = subprocess.Popen(args, stdout=subprocess.PIPE).communicate()
+        stdout, stderr = subprocess.Popen(
+            args, stdout=subprocess.PIPE, shell=sys.platform == "win32"
+        ).communicate()
```

Comment:
This would probably make more sense with direct support for this in grass.script, but to limit the solution only to here, would shutil.which work here at line 22?